move-mutation-test/src/main.rs

```diff
@@ -4,24 +4,51 @@
 
 #![forbid(unsafe_code)]
 
-use clap::Parser;
+use clap::{Parser, Subcommand};
 use move_mutation_test::{
     cli::{CLIOptions, TestBuildConfig},
     run_mutation_test,
 };
+use std::path::PathBuf;
 
 #[derive(Parser)]
-pub struct Opts {
-    /// Command line options for the mutation tester.
-    #[clap(flatten)]
-    pub cli_options: CLIOptions,
-    /// The configuration options for running the tests.
-    #[clap(flatten)]
-    pub test_build_config: TestBuildConfig,
+#[command(version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct Opts {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Runs the mutation test tool.
+    Run {
+        /// Command line options for the mutation tester.
+        #[clap(flatten)]
+        cli_options: CLIOptions,
+        /// The configuration options for running the tests.
+        #[clap(flatten)]
+        test_build_config: TestBuildConfig,
+    },
+
+    /// Display the report in a more readable format.
+    DisplayReport {
+        /// Report location. The default file is "report.txt" under the same directory.
+        #[clap(short = 'p', long)]
+        path_to_report: Option<PathBuf>,
```

Comment:
Maybe we could use PathBug with some default value already here, in `clap` parser - it would be then less code with checking and unwrapping option - it's only a❓

Reply:
Let's keep it simple.
Done